src/ches/emailHandlersPublic.ts

```diff
@@ -103,54 +111,54 @@ export const sendProvisionedEmails = async (request) => {
           ]
             .filter(Boolean)
             .map(({ email }) => email),
-          from: "Registry <cloud.pathfinder@gov.bc.ca>",
-          subject:
-            "Your product change request has been provisioned",
+          from: 'Registry <cloud.pathfinder@gov.bc.ca>',
+          subject: 'Your product change request has been provisioned',
         });
       }
-
     }
   } catch (error) {
     console.error(error);
   }
 };
 
-
 export const sendEditRequestEmails = async (editRequest) => {
-  try {
-    await chesService.send({
-      bodyType: "html",
-      body: swig.renderFile(
-        "./src/ches/publicEmailTemplates/edit-request-received.html",
-        generateEmailTemplatePublicData(editRequest.requestedProject, editRequest.project)
-      ),
-      // For all project contacts. Sent when a new quota edit request is submitted successfully.
-      to: [
-        editRequest.project.projectOwner,
-        editRequest.project.primaryTechnicalLead,
-        editRequest.project.secondaryTechnicalLead,
-        editRequest.requestedProject?.projectOwner,
-        editRequest.requestedProject?.primaryTechnicalLead,
-        editRequest.requestedProject?.secondaryTechnicalLead,
-      ]
-        .filter(Boolean)
-        .map(({ email }) => email),
-      from: "Registry <cloud.pathfinder@gov.bc.ca>",
-      subject: `${editRequest.requestedProject.name} edit request received`,
-    });
-  } catch (error) {
-    console.error(error);
+  if (requestedChanges(editRequest.project, editRequest.requestedProject)) {
```

Comment:
What about we returns the function early if there is no changes so that we can avoid indented codebase below in terms of the readability.